lib/generators/install/templates/microservices_engine.rb

```diff
@@ -0,0 +1,22 @@
+require 'net/http'
+config_file = '../mse_router_info.yml'
+puts("[MSE] > No router configuration YAML file found") and return if
```

Comment:
The trailing `if` is only for inline usage - as long as you're introducing a line break you may as well write this in standard `if` notation.

Also, `puts and return` is basically a `raise` right?